sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1008,6 +1008,16 @@ public enum SkuName
         StandardZrs = 3,
         PremiumLrs = 4,
     }
+    public partial class StartCopyFromUriOptions
```

Comment:
Should this just be `CopyFromUriOptions`?  The `Start` prefix is only to signify that it's an LRO and not part of the operation name.  I'll ping @KrzysztofCwalina to see if he has a preference and get back to you ASAP.

Reply:
I'd vote for ```CopyBlobFromUriOptions```, it's less generic.

Reply:
Krzysztof is in favor of shorter/simpler names when we can make sure we're not going to end up with conflicts down the road.  I'm not worried about adding a `CopyFromUriAsync` that would take different parameters so that's fine.

I think you bring up a really good point that we may possibly end up adding other copy operations in the future.  How about `BlobCopyFromUriOptions` since other copy operations would probably be operating on blobs and the pattern we've been going with is `[Operation]Options` when possible or `[FooClient - Client][Operation]Options`?  This would leave us room for `(BlobContainer/Service/????)CopyFromUriOptions` if ever needed.

Does that sound reasonable?  Ping me if you want to chat more to hopefully unblock this faster.

Reply:
I think ```BlobCopyFromUriOptions``` sounds good.